.github/workflows/ci_webui.yaml

```diff
@@ -38,6 +38,11 @@ jobs:
           npm run format
           git diff --exit-code
 
+      - name: Regenerate translation source
```

Comment:
Why this step?

Reply:
To make sure the en.json contains the latest up to date strings, for syncing to Transifex.
Let me know if it isn't required now (after your latest changes).

Reply:
Removed.